packages/tables/resources/views/columns/layout/stack.blade.php

```diff
@@ -15,9 +15,10 @@
             ->class([
                 'flex flex-col',
                 match ($alignment) {
-                    Alignment::Start => 'items-start',
+                    Alignment::Start, Alignment::Left => 'items-start',
                     Alignment::Center => 'items-center',
                     Alignment::End, Alignment::Right => 'items-end',
+                    Alignment::Justify => null,
```

Comment:
`Alignment::Between` should probably also be handled.
Alternatively, we could decide to not support custom classes for alignment and just do `default => null`?

Reply:
i think that would be a breaking change as some might do `->align('justify-between')` and include that php file in vite? as currently you can't `->alignBetween`

however I do agree to just do a `default => null`

Reply:
Fair, that could be breaking indeed.
Right, @danharrin?

Reply:
Yeah, breaking